qiskit/primitives/statevector_estimator.py

```diff
@@ -35,6 +35,61 @@ class StatevectorEstimator(BaseEstimatorV2):
     pure state vectors. These states are subsequently acted on by :class:~.SparsePauliOp`,
     which implies that, at present, this implementation is only compatible with Pauli-based
     observables.
+
+    Each tuple of ``(circuit, observables, <optional> parameter values, <optional> precision)``,
+    called an estimator primitive unified bloc (PUB), produces its own array-based result. The
+    :meth:`~.EstimatorV2.run` method can be given many pubs at once.
+
+    .. code: python
+
+        from qiskit.circuit import Parameter, QuantumCircuit
+        from qiskit.primitives import StatevectorEstimator
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+
+        # Define a circuit with two parameters.
+        circuit = QuantumCircuit(2)
+        circuit.h(0)
+        circuit.cx(0, 1)
+        circuit.ry(Parameter("a"), 0)
+        circuit.rz(Parameter("b"), 0)
+        circuit.cx(0, 1)
+        circuit.h(0)
+
+        # Define a sweep over parameter values, where the second axis is over
+        # the two parameters in the circuit.
+        params = np.vstack([
+            np.linspace(-np.pi, np.pi, 100),
+            np.linspace(-4 * np.pi, 4 * np.pi, 100)
+        ]).T
+
+        # Define three observables. Many formats are supported here including classes
+        # such as qiskit.quantum_info.SparsePauliOp. The inner length-1 lists cause this array
+        # of observables to have shape (3, 1), rather than shape (3,) if they were omitted.
+        observables = [["XX"], [{"IX": 0.5, "IY": 0.5}], ["ZZ"]]
```

Comment:
Might be better to use the qi classes in this example since strings are semi-internal representations atm
```suggestion
        observables = [[qi.Pauli("XX")], [SparsePauliOp(["IX", "IY"], [0.5, 0.5])], [Pauli("ZZ")]]
```

Reply:
Actually might be nice to make the example a case where you can run a sparse pauli op, and its individual terms to get back the sum and the compents
```suggestion
        observables = [[SparsePauliOp(["XI", "IY"], [0.5, 0.5])], [Pauli("XI")], [Pauli("IY")]]
```

Reply:
In [5d07bd1](https://github.com/Qiskit/qiskit/pull/11589/commits/5d07bd1094adc171de409b00e9dfe6e20bcd83ba) I switched to quantum_info objects in the examples. 

However, I used XX instead of XI because XI commutes with all of the example's final states.